pyvpgl.cxx

```diff
@@ -112,12 +175,55 @@ void wrap_vpgl(py::module &m)
     .def(py::init<vpgl_lvcs const&, vpgl_rational_camera<double> const&>())
     .def(py::init<double, double, double, vpgl_rational_camera<double> const&>())
     .def("set_lvcs", &vpgl_local_rational_camera<double>::set_lvcs)
-    .def("lvcs", &vpgl_local_rational_camera<double>::lvcs);
+    .def("lvcs", &vpgl_local_rational_camera<double>::lvcs)
+    .def("project",
+      [](vpgl_local_rational_camera<double> &R, const double x, const double y, const double z)
+      {
+        vgl_point_3d<double> world_point(x,y,z);                                 
+        vgl_point_2d<double> img_point = R.project(world_point);
+        return std::make_tuple(img_point.x(), img_point.y());
+      }
+    );
 
   m.def("read_local_rational_camera",
         [](std::string const& fname){return read_local_rational_camera<double>(fname);},
         py::return_value_policy::take_ownership);
 
+  // Compute Affine Camera from Local Rational Camera
+  m.def("compute_affine_from_local_rational", 
```

Comment:
Should this function go into vxl, and then be wrapped?

Reply:
I'm not sure. It is a similar case of this being essentially a copy from the old-style python contrib that was probably a one-off process. I could add it to the local rational cam vxl code if you think that's where it should go though.

Reply:
@wtreible This functionality is now available in vpgl_algo and wrapped in pyvpgl_algo